packages/client/components/promptResponse/StandardBubbleMenu.tsx

```diff
@@ -25,7 +28,12 @@ export const StandardBubbleMenu = (props: Props) => {
         tippyOptions={{duration: 100, role: 'dialog'}}
         shouldShow={shouldShowBubbleMenu}
       >
-        <div className='flex items-center rounded-sm border-[1px] border-solid border-slate-600 bg-white p-[3px]'>
+        <div
+          className={clsx(
```

Comment:
-1 use [`cn` ](https://github.com/ParabolInc/parabol/blob/168ef0bd6a7ecbd435c0e0884330371a3c9d72da/packages/client/ui/cn.ts) to merge class names.  clsx combines multiples and twMerge knows how to dedupe competing tailwind classes. we still have a few cases where our app uses one or the other. i bet AI would be good at refactoring to use `cn`!

Reply:
Updated! I'll create a second PR replacing `clsx` with `cn` in the codebase